core/tests/webdriverio_desktop/userJourneys.js

```diff
@@ -16,27 +16,26 @@
  * @fileoverview End-to-end tests for user management.
  */
 
-var action = require('../protractor_utils/action.js');
+var action = require('../webdriverio_utils/action.js');
 var CollectionEditorPage =
-  require('../protractor_utils/CollectionEditorPage.js');
+  require('../webdriverio_utils/CollectionEditorPage.js');
 var CreatorDashboardPage =
-  require('../protractor_utils/CreatorDashboardPage.js');
+  require('../webdriverio_utils/CreatorDashboardPage.js');
 var ExplorationEditorPage = require(
-  '../protractor_utils/ExplorationEditorPage.js');
-var LibraryPage = require('../protractor_utils/LibraryPage.js');
-var ModeratorPage = require('../protractor_utils/ModeratorPage.js');
-var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
-var forms = require('../protractor_utils/forms.js');
-var general = require('../protractor_utils/general.js');
-var users = require('../protractor_utils/users.js');
-var waitFor = require('../protractor_utils/waitFor.js');
-var workflow = require('../protractor_utils/workflow.js');
+  '../webdriverio_utils/ExplorationEditorPage.js');
+var LibraryPage = require('../webdriverio_utils/LibraryPage.js');
+var ModeratorPage = require('../webdriverio_utils/ModeratorPage.js');
+var PreferencesPage = require('../webdriverio_utils/PreferencesPage.js');
+var forms = require('../webdriverio_utils/forms.js');
+var general = require('../webdriverio_utils/general.js');
+var users = require('../webdriverio_utils/users.js');
+var waitFor = require('../webdriverio_utils/waitFor.js');
+var workflow = require('../webdriverio_utils/workflow.js');
 
 var _selectLanguage = async function(language) {
-  var languageDropdown = element(by.css('.protractor-test-language-dropdown'));
-  var languageOption = element(
-    by.css('.protractor-test-i18n-language-' + language));
+  var languageDropdown = $('.e2e-test-language-dropdown');
   await action.click('Language Dropdown', languageDropdown);
+  var languageOption = $('.e2e-test-i18n-language-' + language);
```

Comment:
The issue is resolved now.

Reply:
Now resolved because the updated changes is leading to [flakiness.](https://github.com/oppia/oppia/runs/7092787230?check_suite_focus=true#step:12:16383)